app/controllers/webhook_settings_controller.rb

```diff
@@ -20,7 +20,9 @@ def update
     id = params[:webhook_id]
     webhook = Webhook.where(:project_id => @project.id).where(:id => id).first
     webhook.url = params[:url]
-    if webhook.save
+    if webhook.url.empty?
+      webhook.destroy
```

Comment:
Good suggestion!
I didn't realize the need to turn off webhooks.

I think that `webhook.destroy` will rarely fail.
I will add to show error messages in another PR.